unimod_mapper/unimod_mapper.py

```diff
@@ -105,9 +106,10 @@ def _parseXML(self, xml_file_list=None):
         if xml_file_list is None:
             xml_file_list = []
         data_list = []
-        for xml_path in xml_file_list:
-            xml_path = Path(xml_path)
-            if os.path.exists(xml_path):
+        usermod_id = 10000
```

Comment:
I don't think it's the best idea to just number usermod_ids starting at 10000. The unimod IDs may very well reach >10000 at some point. It could also lead to confusions about what unimods are and what userdefined mods are. That's why we used to give them are clearly different ID ("u1", "u2", etc).

Reply:
Good idea